packages/enhanced/src/lib/container/runtime/CustomRuntimeModule.ts

```diff
@@ -0,0 +1,65 @@
+import { normalizeWebpackPath } from '@module-federation/sdk/normalize-webpack-path';
+import { getFederationGlobalScope } from './utils';
+import { transformSync } from '@swc/core';
+
+const { RuntimeModule, Template, RuntimeGlobals } = require(
+  normalizeWebpackPath('webpack'),
+) as typeof import('webpack');
+
+const federationGlobal = getFederationGlobalScope(RuntimeGlobals);
+
+class CustomRuntimeModule extends RuntimeModule {
+  private entryModuleId: string | number | undefined;
+
+  constructor(
+    private readonly bundledCode: string | null,
+    entryModuleId: string | number | undefined,
+  ) {
+    super('CustomRuntimeModule', RuntimeModule.STAGE_BASIC);
+    this.bundledCode = bundledCode;
+    this.entryModuleId = entryModuleId;
+  }
+
+  override identifier() {
+    return 'webpack/runtime/embed/federation';
+  }
+
+  override generate(): string | null {
+    if (!this.bundledCode) return null;
+    const { code: transformedCode } = transformSync(this.bundledCode, {
```

Comment:
the comments of webpack code gen have problems when combined in another runtime with same comments in runtime - this removes the comments - in the future i think it will be possible to skip emitting modules to disk and instead adding them to a chunk of the parent compile - similar to mini css